app/components/parts/Button/index.ts

```diff
@@ -0,0 +1 @@
+export {Button} from './Button';
```

Comment:
Needs a newline.

Reply:
@HonkingGoose 
I've made corrects
Would merging this commit enable renovate to function properly?

Reply:
I think you're good to go on the Renovate front. 😉

Reply:
Thank you for your support, I appreciate it.